.github/workflows/ci.yaml

```diff
@@ -159,7 +159,7 @@ jobs:
         uses: actions/cache@v4
         with:
           path: /tmp/pip-cache
-          key: nox-${{ matrix.nox-session }}-${{ matrix.arch }}-${{ matrix.os }}-${{ matrix.python }}-${{ hashFiles('**/pyproject.toml') }}
+          key: nox-${{ matrix.nox-session }}-${{ matrix.arch }}-${{ matrix.os }}-${{ matrix.python }}-${{ hashFiles('/pyproject.toml') }}
```

Comment:
I think this needs to remove the slash `/` too as it was done for https://github.com/frequenz-floss/frequenz-client-dispatch-python/commit/685c90a6d398799c2baf055fd4d5e417d66dac5f